signs-work-order-map/components/Map.tsx

```diff
@@ -82,19 +105,24 @@ export default function Map({ location, signs }: MapProps) {
       {signPins}
 
       {
-        // showing the location / geolocation will happen in a subsequent issue
-        console.log("location from payload: ", location)
-        // when do we show location vs signs? when there are no signs?
-        /* {location?.latitude && location?.longitude && (
-        <Marker latitude={location.latitude} longitude={location.longitude} />
-      )} */
+        // the add location marker, work will be completed in a following PR
+        // location?.latitude && location?.longitude && (
+        //   <Marker latitude={location.latitude} longitude={location.longitude} />
+        // )
+        console.log(location)
```

Comment:
if we're going to keep this log around, would you add some more detail/context to the message?